identity/src/main.rs

```diff
@@ -1,59 +1,33 @@
-use std::{io, process, sync::Arc};
-
-use diesel::r2d2::{ConnectionManager, Pool};
-use dotenv::dotenv;
-use envconfig::Envconfig;
+use std::{io, sync::Arc};
 
 #[macro_use]
 extern crate diesel_migrations;
 
-use identity::config::Configuration;
-use identity::db::Db;
+use identity::config::{configuration, Configuration};
+use identity::db::{db_pool, DbPool};
 use identity::rpc::rpc_server;
```

Comment:
```suggestion
use identity::{config::configuration, db::db_pool, rpc::rpc_server};
```

Reply:
Works in combination with belows changes.

Reply:
This is exactly the point why I like the hierarchical import more, it is clearer where things should go. With this approach I always have the problem if I should split it in several lines or not. Nonetheless, I will fix it.